scripts/_common.sh

```diff
@@ -37,16 +37,28 @@ install_jellyfin_packages() {
 	ynh_setup_source --dest_dir="$tempdir" --source_id="ffmpeg_$debian"
 	ynh_setup_source --dest_dir="$tempdir" --source_id="server_$debian"
 
+
+	# Install the packages
+	ynh_package_install \
+		"$tempdir/jellyfin-web.deb" \
+		"$tempdir/jellyfin-server.deb"
+
 	# We need to workaround yunohoost passing --no-remove to replace jellyfin-ffmpeg5...
 	if ynh_package_is_installed "jellyfin-ffmpeg5"; then
 		ynh_package_remove "jellyfin-ffmpeg5"
 	fi
 
 	# Install the packages
 	ynh_package_install \
-		"${tempdir}/jellyfin-web.deb" \
-		"${tempdir}/jellyfin-server.deb" \
-		"${tempdir}/jellyfin-ffmpeg.deb"
+		"${tempdir}/jellyfin-ffmpeg6.deb"
+
+	# The doc says it should be called only once,
+	# but the code says multiple calls are supported.
+	# Also, they're already installed so that should be quasi instantaneous.
+	ynh_install_app_dependencies \
```

Comment:
Why 2 install steps? Can we not just choose one?  It was working with just one...

Reply:
Apparently they don't do the same stuff and we should leave it like that